NorwegianRefugeeCouncil/core
# [CORE24-158] feat(participants): Endpoint for getting a participant by ID

Jira ticket: JIRA-CORE24-158


## What have you implemented?

Endpoint `/api/participants/:id` retrieves the participant from the database and returns it.
Set the server timezone to UTC.

## How to test it?

Create a participant, then call the new endpoint with its ID. Expect the values to be as they were defined in the creation.

## Extra/next steps

Missing tests for the participant store. Will create an additional PR including integration tests for it, instead of unit tests as currently defined.

Add `TZ=UTC` to your local `.env` file if you want to run tests successfully ;)
